src/Components/Tailwind/Toggle.tsx

```diff
@@ -61,13 +61,21 @@ export const Toggle = ({
   );
 };
 
-interface CheckinToggleProps {
+interface CheckStateToggleProps {
   checked: boolean;
   isLoading: boolean;
+  checkState: string;
```

Comment:
```suggestion
  checkInState: string;
```

Here also (consistency).

Reply:
I didn't add the "In" here because the toggle can now be used to toggle Check-out state, that's why I renamed it

Reply:
yeah, it's the same in the indico PR... tbh, I'm not the biggest can of calling it "checks" instead of "checkins" (even if it may be a "leaving checkin" aka "checkout")

Reply:
I can certainly rename it there as well, I couldn't come up with a good name for this. Btw I left the part that has to do with the check-in app api still called check in